### ISSUE TITLE
Peek in node modules directories when grouping

### ISSUE THREAD

**Issue by lencioni** [trotzig/import-js#208]
If you depend on a package `foo` but end up importing a file that exists within the package, such as `foo/some/other/thing`, you might write:

```js
import thing from 'foo/some/other/thing';
```

In this case, we not currently resolving this as a package dependency when grouping, but we should.

**Comment by trotzig** [comment]
Yeah, I've seen this too with some aliases that we have:

```json
    "branch": "imports?define=>false!branch-sdk",
```

Your example could potentially be solved by "peeking" into the node_modules dir. It could potentially also be enough to match the start of the import path (up until the first `'/'`). But my aliases is left in the cold with that strategy. I wonder if we should consider adding a property to aliases so that you can specify if they are a "package dependency" or not?

**Comment by trotzig** [comment]
For now, I think I'm going to implement the "up until the first slash" solution.

**Comment by lencioni** [comment]
Ok, I think that sounds fine.

As for aliases, I think there might be something funny. In a project where I have a jquery alias like:

```
"$": "jquery"
```

and I have jquery as a dependency in my package.json, I found it sometimes sorting it with the other packages, and sometimes in its own group. I think it depended on where it was when I ran the command, but I wasn't able to figure out a clear pattern. It might be worth digging.
